flask_project/campaign_manager/templates/components/project_header.html

```diff
@@ -10,7 +10,27 @@ <h1>{{ name }}</h1>
             <path fill="#D73F3F" fill-rule="nonzero" d="M13.79 6.786a1.202 1.202 0 0 0-1.705 0l-.427.429-5.971 6.004.001.001-.188.19s-.6.607-1.955 5a56.6 56.6 0 0 0-.168.55l-.056.187-.13.441c-.098.332-.336 1.08-.067 1.35.259.262 1.012.032 1.341-.066.15-.044.294-.088.436-.131l.194-.059.205-.063.25-.078.074-.022c4.161-1.302 4.908-1.913 4.97-1.968l.001-.001.005-.004.192-.194.013.013 5.972-6.004.426-.429a1.218 1.218 0 0 0 0-1.715l-3.412-3.431zM9.914 17.77a.792.792 0 0 1-.05.032l-.017.01c-.199.12-.782.428-2.197.942-.165.06-.345.124-.534.19L5.104 16.92c.065-.19.129-.373.189-.54.511-1.427.818-2.014.936-2.214l.008-.013a2.448 2.448 0 0 1 .033-.053l.147-.148 3.646 3.666-.15.151zM20.616 6.786l-3.413-3.43a1.202 1.202 0 0 0-1.706 0l-.853.857a1.218 1.218 0 0 0 0 1.715l3.412 3.431a1.202 1.202 0 0 0 1.707 0l.853-.858a1.218 1.218 0 0 0 0-1.715z" />
           </svg>
           Edit</button>
-        <button class="btn btn-primary btn-xs">
+
+        <div id="delete-background" style="display:none;"></div>
+          <div id="delete-pop" style="display:none;">
```

Comment:
We can use the Bootstrap modal here, since the project already has the CSS and JS for it. Here's an example: https://getbootstrap.com/docs/3.4/javascript/#modals-examples

We're currently using this for other modals in the app, like the warning when someone tries to click "Create" before logging in, and the modal when a user adds types during campaign creation.

We'll have to write some CSS overrides to make the modal match the new design. I think those overrides could go in `flask_project/campaign_manager/styles/components/_modal.scss`.

Reply:
OK, thanks! I'll change to using a modal for the popup! :ok_hand: